tests/unit/s2n_resume_test.c

```diff
@@ -642,7 +642,7 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_connection_free(conn));
         }
 
-        /* Deserialized ticket sets correct connection values for session resumption in TLS1.2, without EMS data */
+        /* Deserialized ticket without EMS data errors */
         {
             struct s2n_blob ticket_blob = { 0 };
             EXPECT_SUCCESS(s2n_blob_init(&ticket_blob, tls12_ticket, sizeof(tls12_ticket)));
```

Comment:
Do you think it's worth adding a hardcoded tls12 ticket with your new format + ems? https://github.com/aws/s2n-tls/blob/7862b0b491747bc96868344e48c1e891c3bbda5f/tests/unit/s2n_resume_test.c#L579-L622

Reply:
Eh, this is basically already tested [here](https://github.com/aws/s2n-tls/blob/main/tests/unit/s2n_resume_test.c#L669).

Reply:
It is right now, but I think one of the main points of the hardcoded tickets was to catch changes that would affect decrypting existing tickets. Since they're hardcoded, they're not affected by changes to the serialize code.

Reply:
I added a test for this.